src/main/java/com/textmessenger/controller/MessageController.java

```diff
@@ -24,15 +26,15 @@
 
 
   @GetMapping("/dialog/{id}")
-  public ResponseEntity<?> getAllMessagesByDialog(@PathVariable("id") Dialog dialog) {
+  public ResponseEntity getAllMessagesByDialog(@PathVariable("id") Dialog dialog) {
     return Optional.of(ResponseEntity.ok().body(messageService.getMessagesFromDialog(dialog)))
             .orElse(ResponseEntity.noContent().build());
   }
 
-  @PostMapping("dialog/{id}")
-  public ResponseEntity<?> addMessageToDialog(@PathVariable("id") Dialog dialog, @RequestBody Message message) {
-    message.setDialog(dialog);
-    messageService.createMessage(message);
+  @PostMapping("/user/{userId}/dialog/{id}")
+  public ResponseEntity<?> addMessageToDialog(@PathVariable("userId") Long user,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove usage of generic wildcard type. ([squid:S1452](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1452))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=120&issues=AWUz3g7-W2AOP0CkBtXY&open=AWUz3g7-W2AOP0CkBtXY)

[ISSUE_KEY]: AWUz3g7-W2AOP0CkBtXY